build/manager/Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN mkdir -m 00755 /opt/manager
 # Refer to https://github.com/GoogleContainerTools/distroless for more details
 FROM --platform=$BUILDPLATFORM gcr.io/distroless/static:nonroot
 
-ENV GOTRACEBACK=all
+ENV GOTRACEBACK=single
```

Comment:
What does this do?

Reply:
For the change to `GOTRACEBACK=single` see <https://skaffold.dev/docs/workflows/debug/#go-runtime-go>.